src/Iterators/LDAPIterator.php

```diff
@@ -286,8 +287,8 @@ public function rewind()
         reset($this->entries);
         $this->current = current($this->entries ?? []);
     }
-    #[\ReturnTypeWillChange]
-    public function valid()
+
+    public function valid(): bool
```

Comment:
Probably we should add `: bool` to the `fetchPagedResult()` declaration as well, in this case

Reply:
Done